i18n/it/cosmic_settings.ftl

```diff
@@ -27,212 +27,216 @@ import = Importa
+        [integrated] Disattiva la GPU dedicata per una maggior durata della batteria e meno rumore proveniente dalle ventole.
+        [nvidia] Miglior esperienza grafica ma maggior consumo energetico. { -requires-restart }.
     }
-    .restart = Riavvia e passa alla modalità { $mode }?
-    .restart-desc = Passare alla modalità { $mode } chiuderà tutte le applicazioni aperte
+    .restart = Riavviare in modalità { $mode }?
+    .restart-desc = Il riavvio in modalità { $mode } chiuderà tutte le applicazioni aperte.
 
-mirroring = Duplicazione schermo
+mirroring = Duplicazione
     .id = Duplicazione { $id }
     .dont = Non duplicare
     .mirror = Duplica { $display }
-    .project = Proietta verso { $display ->
+    .project = Proietta su { $display ->
         [all] tutti gli schermi
         *[other] { $display }
     }
-    .project-count = Proiettare verso { $count} { $count ->
-        [1] schermo
-        *[other] schermi
+    .project-count = Proiettamento su { $count} { $count ->
+        [1] altro schermo
+        *[other] altri schermi
     }
 
-night-light = Luce Notturna
-    .auto = Automatica (dall'alba al tramonto)
-    .desc = Riduce la luce blu tramite colori più caldi.
+night-light = Modalità notturna
+    .auto = Automatica (dal tramonto all'alba)
+    .desc = Riduci la luce blu usando colori più caldi.
 
 orientation = Orientamento
-    .landscape = Normale
-    .landscape-flipped = Ruotato di 90°
-    .portrait = Ruotato di 180°
-    .portrait-flipped = Ruotato di 270°
+    .standard = Standard
+    .rotate-90 = 90 gradi in senso orario
+    .rotate-180 = 180 gradi
+    .rotate-270 = 90 gradi in senso antiorario
 
 scheduling = Pianificazione
     .manual = Pianificazione manuale
 
 ## Desktop: Notifications
 
 notifications = Notifiche
-    .desc = Non disturbare, notifiche blocco schermo, e impostazioni specifiche per applicazione.
+    .desc = Non disturbare, notifiche nel blocco schermo e impostazioni per le applicazioni
 
 ## Desktop: Options
 
-desktop-panel-options = Scrivania e Pannello
-    .desc = Azione per il Tasto Super, angoli caldi, opzioni di controllo finestra.
+desktop-panel-options = Desktop e Pannello
+    .desc = Azione tasto Super, bordi reattivi e opzioni controllo finestre.
 
-desktop-panels-and-applets = Pannelli Scrivania e Applet
+desktop-panels-and-applets = Pannello e applet
 
-dock = Barra applicazioni
-    .desc = Pannello con le applicazioni preferite.
+dock = Barra delle applicazioni
+    .desc = Pannello contenente le applicazioni fissate.
 
-hot-corner = Angolo Caldo
-    .top-left-corner = Abilita angolo caldo in alto a sinistra per spazi di lavoro
+hot-corner = Bordi reattivi
+    .top-left-corner = Abilita bordo reattivo in alto a sinistra per gli spazi di lavoro
 
-super-key-action = Azione per il Tasto Super
-    .launcher = Avviatore
+super-key-action = Azione tasto Super
+    .launcher = Launcher
```

Comment:
This got untranslated

Reply:
"Launcher" is another thing that every Italian will understand because of Smartphones.
"Avviatore" is the literal translation of "Launcher" and, in my opinion, is just ridicolous.

The same is for "Scrivania" instead of "Desktop".
"Scrivania" would be fine if we are talking about the physical desktop.
We are not.

Reply:
I disagree that every Italian will understand launcher. As I mentioned in other comments, not everyone using a computer is tech savy.
However I agree that "avviatore" might not be perfect and this was for sure one of the translations I was most torn about, since I didn't find any previous literature on the matter. Decided to go with "avviatore", in the sense of something used to launch (per avviare) applications for accessibility. Also see the notes in [the previous PR](https://github.com/pop-os/cosmic-settings/pull/26). 

Scrivania is another matter, but again it's debatable, since from a quick search you can find many resources using both "scrivania" and "desktop".